### PR TITLE
Add debugger tools to main toolbar

### PR DESCRIPTION
**Pull request by pahans**

### CODE REVIEW

File: modules/web/src/core/toolbar/components/Tool.jsx [hunk 1]
```diff
@@ -12,13 +12,17 @@ class Tool extends React.Component {
      * @inheritdoc
      */
     render() {
-        const { tool: { icon, commandID, commandArgs = {}, isActive = () => true },
+        const { tool: { icon, commandID, commandArgs = {}, isActive = () => true, isVisible = () => true },
                 dispatch } = this.props;
+
+        if (!isVisible()) {
+            return (<noscript />);
```
Comment (dakshika) [inline modules/web/src/core/toolbar/components/Tool.jsx]: Is there any unsupported things here?

Comment (pahans) [reply modules/web/src/core/toolbar/components/Tool.jsx]: `<noscript />` is used when we dont want to render anything in react.

Comment (pahans) [reply modules/web/src/core/toolbar/components/Tool.jsx]: more info https://stackoverflow.com/questions/31231913/does-react-have-a-null-element/31232642

Comment (pahans) [reply modules/web/src/core/toolbar/components/Tool.jsx]: seems they have updated API to use null instead. https://facebook.github.io/react/blog/2014/07/17/react-v0.11.html#rendering-to-null

**Pull request merged**